wolfssl/ssl.h

```diff
@@ -1145,6 +1145,9 @@ WOLFSSL_API int  wolfSSL_set1_sigalgs_list(WOLFSSL* ssl, const char* list);
 #endif
 WOLFSSL_ABI WOLFSSL_API WOLFSSL* wolfSSL_new(WOLFSSL_CTX* ctx);
 WOLFSSL_API WOLFSSL_CTX* wolfSSL_get_SSL_CTX(const WOLFSSL* ssl);
+#ifdef WOLFSSL_LEANPSK_STATIC
+WOLFSSL_API WOLFSSL* wolfSSL_new_leanpsk(WOLFSSL_METHOD* method, byte ciphersuite0, byte ciphersuite1, unsigned char* ran, int ranSz);
```

Comment:
The implementation of this function is in `mplabx/small-psk-build/psk-ssl.c`.  Why are these defined here?